examples/developing_estimators/sklearn_is_fitted.py

```diff
@@ -0,0 +1,72 @@
+"""
+==========================================
+``__sklearn_is_fitted__`` as Developer API
```

Comment:
Nitpick but single backtick is preferred see: https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation second to last bullet

Reply:
@lucyleeow thanks for the review. I am not sure about this double or single backticks as double backticks are already suggested by @adrinjaali and the guidelines for reStructuredTest (`.rst`) files. But if that's the case I'll change it.

Reply:
Agree that it is not standard. See conversation here:  #24368 and https://github.com/scikit-learn/scikit-learn/pull/14817#issuecomment-1237611996

Reply:
Yeah, I already made the change to single backticks. Thank you for the reference.